mmedit/datasets/pipelines/utils.py

```diff
@@ -129,6 +129,14 @@ def random_choose_unknown(unknown, crop_size):
 
 def make_coord(shape, ranges=None, flatten=True):
     """ Make coordinates at grid centers.
+
+    Args:
+        shape (tuple): shape of image.
+        ranges (tuple): range of coordinate value. Default: None.
+        flatten (bool): flatten to (n, 2) or Not. Default: True.
+
+    return:
+        coord (Tensor): coordinates.
```

Comment:
What's the shape of coord

Reply:
Assume shape = h, w
if flatten: coord.shape = [h, w, 2]
else: coord.shape = [h*w, 2]